docs/js-client-sdk.eppojsclient.initialized.md

```diff
@@ -0,0 +1,11 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@eppo/js-client-sdk](./js-client-sdk.md) &gt; [EppoJSClient](./js-client-sdk.eppojsclient.md) &gt; [initialized](./js-client-sdk.eppojsclient.initialized.md)
+
+## EppoJSClient.initialized property
+
+**Signature:**
+
+```typescript
+static initialized: boolean;

```

Comment:
Docs got updated because the way `index.ts` is structured, `initialized` has to be public 🙄